index.xml

```diff
@@ -8390,7 +8382,7 @@ context.setColorClamped(-1, 255, 257);</x:codeblock>
             <p>
               The prose definition of a constructor <span class='rfc2119'>MUST</span>
               either return an IDL value of a type corresponding to the interface
-              or dictionary the <a class='xattr' href='#Constructor'>[Constructor]</a>
+              the <a class='xattr' href='#Constructor'>[Constructor]</a>
               extended attribute appears on, or throw an exception.
             </p>
             <p>
```

Comment:
That's OK.  I was thinking: "The [Constructor] extended attribute must not be specified on an interface with [NoInterfaceObject]." or "The [Constructor] and [NoInterfaceObject] extended attributes must not be specified on the same interface." or something like that.